frontend/src/Form/beta/ModelCardForm.tsx

```diff
@@ -116,3 +103,18 @@ export default function ModelCardForm({
     </Stack>
   )
 }
+
+interface ValidationErrorIconProps {
+  displayLabelValidation: boolean
```

Comment:
What's the point of this prop? If it's `false` then we don't return anything from the component. If we dont want to use the component then it makes more sense to conditionally render the component in the parent (or not use at all), instead of hiding that behaviour inside the component itself. E.g.
```
{ displayLabelValidation && <ValidationErrorIcon step={step} /> }
```